ivy/functional/ivy/activations.py

```diff
@@ -663,16 +663,27 @@ def softsign(
 @to_native_arrays_and_back
 @handle_array_function
 @handle_device_shifting
+@handle_complex_input
 def mish(
-    x: Union[ivy.Array, ivy.NativeArray], /, *, out: Optional[ivy.Array] = None
+    x: Union[ivy.Array, ivy.NativeArray],
+    /,
+    *,
+    complex_mode: Literal["split", "magnitude", "jax"] = "jax",
+    out: Optional[ivy.Array] = None,
 ) -> ivy.Array:
     """
     Apply the mish activation function element-wise.
 
+    If the input is complex we apply the mish function to the input
+    ``mish(x) = x * tanh(log(1+ e^x))``.
+
```

Comment:
I think you'd be fine without this part of the docstring, since that's also what happens when the input is real